src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgaveService.kt

```diff
@@ -148,8 +148,8 @@ class OppgaveService(
 
     private fun lagSøkeuttrykk(oppgavetype: Oppgavetype): Regex? {
         val s = when (oppgavetype) {
-            Oppgavetype.BehandleSak, Oppgavetype.BehandleUnderkjentVedtak -> "EF.+Sak.+50.+"
-            Oppgavetype.GodkjenneVedtak -> "EF.+Sak.+70.+"
+            Oppgavetype.BehandleSak, Oppgavetype.BehandleUnderkjentVedtak -> "50 Tilbakekreving?.+"
```

Comment:
Kunne denne hackete delen av klassen blitt trukket ut i en util e.l for å prøve å holde servicen litt "ren og pen" ?